pallets/msa/src/tests.rs

```diff
@@ -1,25 +1,26 @@
+use frame_support::{
```

Comment:
We should have an explicit replay rejection test for `add_key_to_msa`

Reply:
I didn't think too hard about this but should we register both signatures?  Is there an attack where I replay one signature with a different second signature?